copier/extensions.py

```diff
@@ -0,0 +1,102 @@
+"""Jinja2 extension to add to the Jinja2 environment."""
```

Comment:
[fix]
```suggestion
"""Jinja2 extension to add to the Jinja2 environment."""
```

better description?

and change file name to jinja_ext.py